build_test/README.md

```diff
@@ -52,6 +52,9 @@ This package will automatically create `*.debug.html` files next to all your
 `*_test.dart` files, which can be loaded in a browser from the normal
 development server (`pub run build_runner serve`).
 
+You may also view an index of links to every `*.debug.html` file by navigating
+to `http://localhost:8081` (or wherever your `test` folder is being served).
```

Comment:
Ah, okay.

Does `pbr test -- -p chrome` print this URI?

Can I "pause after finish" to rerun tests without the server being torn down?
Can I skip running tests all together and just build and host the server so I run tests one-off?

Reply:
> Does `pbr test -- -p chrome` print this URI?

No, but I didn't want to modify PBR in this PR.

> Can I "pause after finish" to rerun tests without the server being torn down?

No idea what that means.

> Can I skip running tests all together and just build and host the server so I run tests one-off?

Yup, you can just use `pub run build_runner serve` to view this.

Reply:
> > Yup, you can just use pub run build_runner serve to view this.

Ah! I expanded the section and see this is under "debugging" – using `pbr serve`

I think this was added to `pbr test`

Nevermind 😄